src/Scrutor/LifetimeSelector.cs

```diff
@@ -1,8 +1,8 @@
-﻿using System;
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyModel;
+using System;
 using System.Collections.Generic;
 using System.Reflection;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.DependencyModel;
```

Comment:
I'd like to avoid these namespaces sorting changes. They don't seem necessary. That goes for the other files as well.

Reply:
done